pkg/users/dialog-utils.js

```diff
@@ -25,7 +25,7 @@ import { show_modal_dialog } from "cockpit-components-dialog.jsx";
 const _ = cockpit.gettext;
 
 export function Validated({ errors, error_key, children }) {
-    const error = errors && errors[error_key];
+    const error = errors?.[error_key];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18448-20230312-091152-46c0aa82-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)